.github/workflows/e2e-detox-template.yml

```diff
@@ -123,7 +128,7 @@ jobs:
     if: ${{ inputs.run-ios-tests }}
     runs-on: macos-14
     continue-on-error: true
-    timeout-minutes: 40
+    timeout-minutes: ${{ inputs.low_bandwidth_mode && 80 || 40 }} 
```

Comment:
Since the network connectivity is slower with defaulted of 1s round trip time, I opted to add 40 more minutes to allow the tests to complete.